ui.apps/src/main/content/jcr_root/apps/hclecomm/config.publish/com.hcl.ecomm.core.services.impl.ProductServiceImpl.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OsgiConfig"
+    cartFetch.servicePath.string="/us/V1/guest-carts/"
+    productService.domainName="localhost:8081/magento/rest"
+    productService.searchCriteriaField="store_id"
+    productService.searchCriteriaValue="4"
```

Comment:
@DeepaliAgrwl : what does searchCriteriaValue="4" signify ?

Reply:
Rishabh this is the store Id for the hclecomm store we created on magento